CHANGELOG.md

```diff
@@ -1,5 +1,49 @@
 # Version changelog
 
+## [Release] Release v0.230.0
+
+Important changes:
+
+Workspace paths are automatically prefixed with `/Workspace`. In addition, all usage of path strings such as `/Workspace/${workspace.root_path}/...` in bundle configuration is automatically replaced with `${workspace.root_path}/...` and generates a warning as part of bundle validate.
```

Comment:
nit: This should be future tense instead of present.
```suggestion
Workspace paths will now automatically be prefixed with `/Workspace`. In addition, all usage of path strings such as `/Workspace/${workspace.root_path}/...` in bundle configuration will automatically replaced with `${workspace.root_path}/...` and will generate a warning as part of ` databricks bundle validate`.
```

Reply:
I think it actually should be a present tense, because it's a present state of things for released version

Reply:
Fair enough. Makes sense. In either case the intent will get across.